UIKitTemplate/UIKitTemplate/Views/SelectorCoffeView.swift

```diff
@@ -0,0 +1,274 @@
+        button.setImage(UIImage(named: "arrow"), for: .normal)
+        button.backgroundColor = UIColor(named: "buttonBackColor")
+        button.layer.cornerRadius = 22
+        return button
+    }()
+
+    /// Кнопка шара промокда
+    let rightNavBarButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 0, y: 0, width: 21, height: 21))
+        button.setImage(UIImage(named: "shareImage"), for: .normal)
+        return button
+    }()
+
+    /// Лейбл с ценой
+    var priceLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 15, y: 669, width: 345, height: 30))
+        label.text = Constants.initOfPriceLabelText
+        label.font = .init(name: "Verdana-Bold", size: 18)
+        label.textAlignment = .right
+        return label
+    }()
+
+    /// Сегмент контроллер выбора кофе
+    private let coffeSegmentController = {
+        let segmentController = UISegmentedControl(items: Constants.selectorLabels)
+        var attribute = [NSAttributedString.Key.font: UIFont(name: "Verdana", size: 13.0)]
+        UISegmentedControl.appearance().setTitleTextAttributes(
+            attribute as [NSAttributedString.Key: Any],
+            for: UIControl.State.normal
+        )
+        segmentController.selectedSegmentIndex = 0
+        segmentController.frame = CGRect(x: 15, y: 368, width: 345, height: 44)
+        segmentController.setDividerImage(
+            UIImage(named: "segmentControllerColor"),
+            forLeftSegmentState: .selected,
+            rightSegmentState: .normal,
+            barMetrics: .default
+        )
+        for subview in segmentController.subviews {
+            subview.layer.shadowOpacity = 0
+            subview.layer.shadowRadius = 0
+        }
+        return segmentController
+    }()
+
+    /// Надпись "Модификация"
+    private let modificationLabel = {
+        let label = UILabel(frame: CGRect(x: 15, y: 432, width: 200, height: 30))
+        label.text = Constants.modificationTextLabel
+        label.font = .init(name: "Verdana-Bold", size: 18)
+        return label
+    }()
+
+    /// Фон для изображения с кофе
+    private let backgroundCoffe: UIImageView = {
+        let view = UIImageView(frame: CGRect(x: 0, y: 0, width: 375, height: 346))
+        view.image = UIImage(named: "backgroundCoffe")
+        return view
+    }()
+
+    /// Кнопка "Заказать"
+    private let makeOrderButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 15, y: 717, width: 345, height: 53))
+        button.setTitle(Constants.makeOrderButtonLabel, for: .normal)
+        button.backgroundColor = UIColor(named: "orderButtonColor")
+        button.titleLabel?.font = .init(name: "Verdana-Bold", size: 18)
+        button.layer.cornerRadius = 12
+        return button
+    }()
+
+    /// Вью с кофе
+    private var coffeView: UIImageView = {
+        let view = UIImageView(frame: CGRect(x: 112, y: 128, width: 150, height: 150))
+        view.image = UIImage(named: "americano")
```

Comment:
```suggestion
        view.image = UIImage(named: "americano")
```
"americano"в  константу